src/components/MarkdownProvider/components/Table.tsx

```diff
@@ -18,7 +19,9 @@ import {
 
 const TableHeadContext = createContext(false);
 
-export const Table = GardenTable;
+export const Table = styled(GardenTable)`
+  margin: 0 0 ${p => p.theme.space.lg};
```

Comment:
Should just be able to use `margin-bottom`. @allisonacs designs for prop tables were to use XL (40px) bottom spacing. Should we be consistent here?